osu.Game/Screens/Multi/Components/ParticipantCount.cs

```diff
@@ -62,6 +62,8 @@ public ParticipantCount()
                     Font = @"Exo2.0-Light"
                 },
             };
+
+            Max = null;
```

Comment:
rather than doing this, i'd prefer you move the update code into an `updateMax` method and just call that. and probably add a value equality check int he property setter to early return on equality and disallow this.